desktop/mac/index.md

```diff
@@ -174,19 +174,17 @@ Select **Use the new Virtualization framework** to allow Docker Desktop to use t
 
 #### Enable VirtioFS
 
-Select the **Enable VirtioFS accelerated directory** sharing option to enable VirtioFS. You must install Docker Desktop 4.6 or higher and macOS 12.2 to use VirtioFS.
+The 4.6 release of Docker Desktop for Mac lets developers use a new experimental file-sharing implementation called [virtioFS](https://virtio-fs.gitlab.io/){: target='_blank' rel='noopener' class='_'}; the current default is gRPC-FUSE. During testing, virtioFS has reduced the time taken to sync changes between the host and VM up to 90%. For more details, see our < need link for 4.6 blog post>.
```

Comment:
Thinking about this now I would actually change:

"During testing, virtioFS has reduced the time taken to sync changes between the host and VM up to 90%. For more details, see our < need link for 4.6 blog post>."

TO

"virtioFS has been found to significantly improve file sharing performance on macOS. For more details, see our blog post here <link to 4.6 blog post>."

Reply:
@christophermclellan SGTM. Thanks.

Reply:
Thank you @christophermclellan Updated